Database/src/main/resources/changelog1.xml

```diff
@@ -838,4 +838,30 @@
         </createTable>
         <addPrimaryKey tableName="tbl_tenant_config" columnNames="tenant_id, key" constraintName="tenant_specific_key" />
     </changeSet>
+    
+    <changeSet author="CAF" id="adding_repository_table-1">
+        <addColumn tableName="tbl_global_config">
+            <column name="scope" type="smallint">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
```

Comment:
I agree with `nullable="false"` but doesn't this also need a default or what value does it put in for preexisting rows?

Reply:
Added and tested locally. All seems to be working (screenshot attached to the Jira)